polkadot/runtime/parachains/src/assigner_on_demand/tests.rs

```diff
@@ -707,3 +707,39 @@ fn queue_status_size_fn_works() {
 		assert_eq!(OnDemandAssigner::get_queue_status().size(), 4)
 	});
 }
+
+#[test]
+fn add_revenue_info_works() {
+	let alice = 100u64;
+	let amt = 10_000_000u128;
+	new_test_ext(GenesisConfigBuilder::default().build()).execute_with(|| {
+		Balances::make_free_balance_be(&alice, amt);
+		assert_eq!(Balances::total_issuance(), amt);
+
+		// Revenue should be empty on block 0
+		assert_eq!(Revenue::<Test>::get().len(), 0);
```

Comment:
Try to use the public interface in tests as much as possible. The less you rely on implementation, the less your test will break in the future.